### PR TITLE
re-synched, updated json reports with "patient" name

### PR DESCRIPTION
**Pull request by sglanger**
Marc; I think you should be able to use this

### CODE REVIEW

**Comment by shadowdoc** [comment]
Steve - when I look at this pull request it only includes one commit, and that commit deletes two files.  One of which, BreastDx-01-0003/MedicationPrescription/levothyroxine.medication_prescription.json was created by me and should be a valid JSON object for this patient.  The original levothyroxine JSON has been moved to the medication folder under the dataset at large.  

Please help me if I'm horribly confused.

**Comment by sglanger** [comment]
levothyroxine.medication_prescription.json was not deleted it was renamed medication_prescription..breastdx.json


to be consistent with all the other json files you will find in the DxReports and other folders

It would probably be simpler if I just mailed them to you. Here they are


________________________________

**Comment by shadowdoc** [comment]
Fine if you want to e-mail them, but they can't be attached through github.

If you look at this URL ->
https://github.com/shadowdoc/siim-dataset/pull/3/files

There are just two deletions in your commit, there are no additions.

--Marc

**Comment by shadowdoc** [comment]
You probably just forgot to add the new files.  Since they weren't being tracked by git they need to be specifically added.  If you just git add, and then git push to your repo they should be added to this pull request automagically.

**Comment by sglanger** [comment]
um, I did a

"git commit -a"


________________________________

**Comment by shadowdoc** [comment]
http://git-scm.com/docs/git-commit says:

-a
--all
Tell the command to automatically stage files that have been modified and deleted, but new files you have not told Git about are not affected.

I've always found git add -p much more useful as you can tell exactly what's being added to the stage.

**Comment by sglanger** [comment]
OK, try it now

**Comment by shadowdoc** [comment]
Reopening this pull request

File: BreastDx-01-0003/AllergyIntolerance/allergy_intolerance.breastdx.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// allergy_intolerance.example.json
+// id: breastdx
```
Comment (shadowdoc) [inline BreastDx-01-0003/AllergyIntolerance/allergy_intolerance.breastdx.json]: This is the ID for the allergy_intolerance - should be set to random as it's unlikely to be referenced elsewhere

**Comment by shadowdoc** [comment]
Will merge, thanks!

**Comment by shadowdoc** [comment]
Also - git mv is your friend when renaming.

**Comment by sglanger** [comment]
thanks

Are you getting together the histories of the other patient's?

________________________________

**Comment by shadowdoc** [comment]
Yes, working on it.

I've got Matt Morgan working on reports for the breast patient.  I've
downloaded the images for the other patients and will work on building
their clinical histories.  Jason has started to contribute as well.

--Marc

**Pull request merged**